src/TileOrientationType.java

```diff
@@ -0,0 +1,3 @@
+public enum TileOrientationType {

```

Comment:
Love the semantics of this I think these will totally work. One thing, can we change this to TileOrientation?